jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss

```diff
@@ -32,15 +32,26 @@
         @extend %floating-panel;
         display: inline-block;
         margin-left: 10px;
+
+        .button--inline-deployment {
+            padding-left: 10px;
+            padding-right: 10px;
+
+            &:first-child {
+                border-top-right-radius: 0;
+                border-bottom-right-radius: 0;
+            }
+
+            &:last-child {
+                border-top-left-radius: 0;
+                border-bottom-left-radius: 0;
+            }
+        }
     }
 
-    &__install-button {
-        @extend %fade-in;
-        @extend %floating-panel;
-        display: inline-block;
-        margin-left: 10px;
-        padding: 10px;
-        color: $cool-grey;
+    .button--inline-neutral {
```

Comment:
All the overriding styles were originally in the generic-button component styles but I felt that if another component was going to modify the generic button then the styles belong to that component. Also it seems more likely that if a developer sees `.deployment-bar .button-deployment {...}` in the inspector then they would go to the deployment-bar component first? 

That was my rationale behind the change. What do you think? Easy to change back if that is the consensus.